portal-impl/src/com/liferay/portal/dao/jdbc/util/DBInfoUtil.java

```diff
@@ -48,11 +51,20 @@ public static DBInfo getDBInfo(DataSource dataSource) {
 					DatabaseMetaData databaseMetaData =
 						connection.getMetaData();
 
+					String jdbcUrl = databaseMetaData.getURL();
+
+					if (keyDataSource instanceof HikariDataSource) {
```

Comment:
@ZoltanTakacs Why this is needed? Is it possible that the url you get from database meta data is not accurate?

Reply:
hey @tinatian , I'm glad you challenged this one. Actually this was on oversight from my side, the using 
 `databaseMetaData.getURL()` is enough.
I pushed this change. 

Thank you! 👍 💯